tests/features/multilingual-aliases.feature

```diff
@@ -0,0 +1,39 @@
+@api
+Feature: Webtools Analytics multilingual aliases
+  In order to identify separate sections of my website
+  As the site manager
+  I need to be able to create rules that allow to pair section names with regular expressions with supporting aliases
+
+  Background:
+    Given I am logged in as a user with the "administer site configuration" permission
+    And the Webtools Analytics configuration is set to use the id '123' and the site path 'sitePath'
+    And the following languages are available:
+      | languages |
+      | en        |
+      | fr        |
+      | nl        |
+    And Aliases available for the path "/admin/config":
```

Comment:
This is a bit un-natural step for a site builder. We should instead have the translations of this content created with the appropriate alias set.

Reply:
Doing this is way too costly as we don't have content types in this module. Let's keep it like this for now.